fmskill/model/point.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import mikeio
+import pandas as pd
+
+from fmskill import types, utils
+from fmskill.comparison import PointComparer, SingleObsComparer
+from fmskill.model import protocols
+from fmskill.model._base import ModelResultBase
+from fmskill.observation import PointObservation, TrackObservation
+
+
+class PointModelResult(ModelResultBase):
+    def __init__(
+        self,
+        data: types.PointType,
+        x: float = None,
+        y: float = None,
+        item: str = None,
+        itemInfo=None,
+        name: str = None,
+        quantity: str = None,
+    ) -> None:
+
+        assert isinstance(
```

Comment:
This check fails on Python 3.8 (not on 3.11), not sure on versions in between.

The reason is that the types is expressed as a Union
```python
PointType = Union[str, Path, pd.DataFrame, mikeio.Dfs0]
```
Which is not understood by `isinstance()` in 3.8
If you use a tuple instead it works as expected for this check, but can not be used as a type hint, since it would indicate that the expected type is a tuple, which is incorrect.
```python
PointTypeTuple = (str, Path, pd.DataFrame, mikeio.Dfs0)
```

Reply:
Yes, and since this is a library which we would like to get broadly adopted, we have to accept a bit of ugly code and some repetition to support a wide range of Python versions. If this was an application, we would do it differently.

I also like to keep things in one place. Makes it easier to be consistent.